src/Raven.Server/ServerWide/DebugInfoPackageUtils.cs

```diff
@@ -57,18 +57,21 @@ public static IEnumerable<RouteInformation> GetAuthorizedRoutes(RavenServer.Auth
                     case RavenServer.AuthenticationStatus.ClusterAdmin:
                         authorized = true;
                         break;
+
                     case RavenServer.AuthenticationStatus.Operator:
                         if (route.AuthorizationStatus != AuthorizationStatus.ClusterAdmin)
                             authorized = true;
                         break;
+
                     case RavenServer.AuthenticationStatus.Allowed:
                         if (route.AuthorizationStatus == AuthorizationStatus.ClusterAdmin || route.AuthorizationStatus == AuthorizationStatus.Operator)
                             break;
                         if (route.TypeOfRoute == RouteInformation.RouteType.Databases
-                            && (db == null || authenticateConnection.CanAccess(db, route.AuthorizationStatus == AuthorizationStatus.DatabaseAdmin) == false))
+                            && (db == null || authenticateConnection.CanAccess(db, requireAdmin: route.AuthorizationStatus == AuthorizationStatus.DatabaseAdmin, requireWrite: route.IsReadOnlyEndpoint) == false))
```

Comment:
` requireWrite: route.IsReadOnlyEndpoint` ?? That seems like the other way around.